app/main.py

```diff
@@ -1,6 +1,23 @@
+import os
+
+import requests
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    base_url = "http://api.weatherapi.com/v1/current.json"
+    api_key = os.environ["API_KEY"]
+    city = "Paris"
+
+    url = base_url + "?key=" + api_key + "&q=" + city
```

Comment:
I won't reject it, but I will leave a suggestion
Using f-strings to form a string with variable is more preferable way